evennia/utils/tests/test_ansi.py

```diff
@@ -8,7 +8,9 @@
 
 from django.test import TestCase
 
-from evennia.utils.ansi import ANSIString as AN
+from evennia.utils.ansi import ANSIString as AN, ANSIParser
```

Comment:
This is not ANSI though, it's Truecolor. I think this belongs in its own test module `utils/tests/test_hex_colors.py`.